.eslintrc.js

```diff
@@ -34,9 +34,11 @@ module.exports = {
 
   overrides: [
     {
-      files: 'test/**/*.js',
-      env: {
-        jest: true
+      files: 'test/typescript/*.ts',
+      rules: {
+        'no-unused-expressions': 'off',
+        '@typescript-eslint/no-namespace': 'off',
+        '@typescript-eslint/no-unused-vars': 'off'
```

Comment:
These rules aren't useful for the TypeScript tests because the code doesn't actually run, it's just meant to be typechecked

Reply:
I might recommend putting a .eslintrc.js file in that path, as it becomes slightly easier to see that those files have overrides applied. Much like we already have a tsconfig.json in that path.

Reply:
Fixed